libs/ui-lib/lib/ocm/components/clusterConfiguration/platformIntegration/ExternalPlatformDropdown.tsx

```diff
@@ -142,6 +154,7 @@ export const ExternalPlatformDropdown = ({
       newFeatureSupportLevelContext,
       featureSupportLevelData,
       cpuArchitecture,
+      isSingleClusterFeatureEnabled,
```

Comment:
this should be added as a dependency to `useEffect`
I see that the eslint rule for checking the dependency array is disabled. I dont see a reason to do so, so please remove it and fix the deps too. Thanks!